iptables/iptables.go

```diff
@@ -63,6 +63,12 @@ func (e ChainError) Error() string {
 
 func initCheck() error {
 	if iptablesPath == "" {
+		if out, err := exec.Command("modprobe", "-va", "nf_nat").CombinedOutput(); err != nil {
```

Comment:
I don't think it should be added to `initCheck`. That gets called only from `Exists`. Why not `NewChain` and this should be done with a once function. Otherwise it is racy and might generate spurious warning messages which might spook users.